packages/webapp/src/containers/Help/index.js

```diff
@@ -6,9 +6,11 @@ import history from '../../history';
 import { userFarmSelector } from '../userFarmSlice';
 export default function HelpRequest() {
   const dispatch = useDispatch();
+  const [loading, setLoading] = useState(false);
 
   const handleSubmit = (file, data) => {
-    dispatch(supportFileUpload({ file, form: data }));
+    setLoading(true);
+    dispatch(supportFileUpload({ file, form: data, setLoading }));
```

Comment:
Passing a function in an action is not a good idea. Actions should be immutable objects. add a loading indicator in the reducer, and get it on the component